lib/ecto/query.ex

```diff
@@ -2191,6 +2191,10 @@ defmodule Ecto.Query do
       function, the function will receive a list of "post_ids" as the argument
       and it must return a tuple in the format of `{post_id, tag}`
 
+  ## Reset preloaded fields
+
+  See `Ecto.reset_fields/2`.
```

Comment:
```suggestion
  If you want to reset the loaded fields, see `Ecto.reset_fields/2`.
```

and we can get rid of the header :)